src/EventEdition/LocationPanel.vala

```diff
@@ -187,6 +189,46 @@ public class Maya.View.EventEdition.LocationPanel : Gtk.Grid {
             debug (error.message);
         }
     }
+
+    private async void find_location (double latitude, double longitude) {
+        if (find_cancellable != null)
+            find_cancellable.cancel ();
```

Comment:
elementaryos code style: Single line if clauses should still have braces.  Also should be followed by blank line.